packages/icons/src/svg/bulb-dualtone.svg

```diff
@@ -0,0 +1,12 @@
+<path fill-rule="evenodd" clip-rule="evenodd" d="M14.3955 8.1016C13.6364 7.67396 12.7767 7.45732 11.9055 7.47414C11.0344 7.49097 10.1837 7.74066 9.44167 8.1973C8.69965 8.65392 8.09336 9.30086 7.68577 10.0709C7.27818 10.841 7.08416 11.7061 7.12382 12.5765C7.16348 13.4469 7.4354 14.2908 7.91133 15.0205C8.05303 15.2379 8.18667 15.4356 8.3151 15.6257C8.32424 15.6392 8.33335 15.6527 8.34244 15.6662C8.83189 16.3903 9.25345 17.014 9.4745 17.8929H14.5313C14.7505 17.0462 15.1559 16.4394 15.6052 15.7669C15.6136 15.7543 15.6221 15.7416 15.6306 15.7289C15.793 15.4857 15.9625 15.2319 16.1428 14.9435C16.6214 14.1776 16.877 13.2935 16.8808 12.3903L16.8808 12.388C16.8868 11.5168 16.6594 10.6598 16.2225 9.906C15.7854 9.15227 15.1546 8.52923 14.3955 8.1016ZM11.8724 5.76018C13.0495 5.73744 14.2112 6.03018 15.2369 6.608C16.2626 7.18583 17.115 8.02769 17.7055 9.04615C18.2957 10.0643 18.6029 11.2218 18.5951 12.3987C18.5897 13.6202 18.2439 14.8161 17.5964 15.8521C17.4014 16.1641 17.2191 16.437 17.058 16.6783L17.0561 16.6811C16.4732 17.5539 16.1875 17.9977 16.0997 18.8391C16.0541 19.2756 15.6861 19.6072 15.2472 19.6072H8.75391C8.31331 19.6072 7.94448 19.2731 7.90096 18.8347C7.81454 17.9639 7.52727 17.5217 6.89468 16.5855L6.89239 16.5821C6.76526 16.394 6.62494 16.1863 6.47539 15.957C5.83231 14.9708 5.4649 13.8306 5.41131 12.6545C5.35772 11.4785 5.61989 10.3095 6.17063 9.26897C6.72136 8.22847 7.54058 7.35431 8.54321 6.73732C9.54585 6.1203 10.6954 5.78292 11.8724 5.76018Z" fill="#101C2D"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M12 0.428589C12.4734 0.428589 12.8572 0.812345 12.8572 1.28573V2.17858C12.8572 2.65197 12.4734 3.03573 12 3.03573C11.5266 3.03573 11.1429 2.65197 11.1429 2.17858V1.28573C11.1429 0.812345 11.5266 0.428589 12 0.428589ZM3.81778 3.81776C4.15251 3.48303 4.69522 3.48303 5.02996 3.81776L5.6613 4.4491C5.99603 4.78383 5.99603 5.32655 5.6613 5.66128C5.32656 5.99602 4.78385 5.99602 4.44911 5.66128L3.81778 5.02994C3.48304 4.69521 3.48304 4.1525 3.81778 3.81776ZM20.1823 3.81778C20.517 4.15252 20.517 4.69523 20.1822 5.02996L19.5509 5.6613C19.2161 5.99602 18.6734 5.99601 18.3387 5.66126C18.0039 5.32652 18.004 4.78381 18.3387 4.44908L18.9701 3.81774C19.3048 3.48302 19.8475 3.48303 20.1823 3.81778ZM0.428467 12C0.428467 11.5266 0.812223 11.1429 1.28561 11.1429H2.17846C2.65185 11.1429 3.03561 11.5266 3.03561 12C3.03561 12.4734 2.65185 12.8572 2.17846 12.8572H1.28561C0.812223 12.8572 0.428467 12.4734 0.428467 12ZM20.9644 12C20.9644 11.5266 21.3481 11.1429 21.8215 11.1429H22.7143C23.1877 11.1429 23.5715 11.5266 23.5715 12C23.5715 12.4734 23.1877 12.8572 22.7143 12.8572H21.8215C21.3481 12.8572 20.9644 12.4734 20.9644 12ZM9.16082 22.7143C9.16082 22.2409 9.54457 21.8572 10.018 21.8572H13.9822C14.4556 21.8572 14.8393 22.2409 14.8393 22.7143C14.8393 23.1877 14.4556 23.5714 13.9822 23.5714H10.018C9.54457 23.5714 9.16082 23.1877 9.16082 22.7143Z" fill="#101C2D"/>
+</g>
+<defs>
+<clipPath id="clip0_10343_43642">
```

Comment:
This shouldn't be part of the generated SVG. It's necessary to clean up vektor in the Figma file.